### DISCUSSION TITLE
Provide ability to remove "close account" and "subscription actions" buttons in user profile.

### DISCUSSION THREAD

**Discussion by Gaploid** [Azure/api-management-developer-portal#2624] [Feature request]
## Bug description
Ability to remove "close account" and "unsubscribe" buttons from user profile. 

## Reproduction steps

1. Go to developer portal
2. Click on sign in 
3. Click to Profile 
4. You see buttons "close account" that is not possible to hid dele

## Expected behavior
Make user profile widget customizable or give ability remove "close account" button and turn off actions for subscriptions.

## Is your portal managed or self-hosted?

Managed

## Additional context
We are controlling the logic of user/subscription provisioning in our app, and the API is part of a broader offering in our service. We don't want users to be enclosed in API management and lose part of the functionality. We provide our own mechanism for canceling a service account in our app and de-provisioning subscriptions in API management if necessary. However, these buttons add complexity to the UX of our service when users could perform similar actions in different parts of the service.

![dev portal cancel button](https://github.com/Azure/api-management-developer-portal/assets/6171245/a6bc9053-e9ec-4e09-9b32-81de87f48d5c)

**Comment by malincrist** [comment]
Hello @Gaploid , thank you for requesting this feature!
I would like to add that you should be able to achieve this behavior by implementing custom widgets that can replace the default ones. You can find more info [here](https://learn.microsoft.com/en-us/azure/api-management/developer-portal-extend-custom-functionality). Hope this helps!

**Comment by davewilksatkeyavi** [comment]
Consider me another user who would like this feature.

**Comment by bendixon-zelis** [comment]
This would be a welcome feature. We ended up writing a custom widget just to get around this. It can be especially detrimental to have a "close account" button when you are using subscription delegation and have no control over the "close account" behavior. It would also be nice if "close account" was considered for delegation so custom flows could be initiated if the user chose to keep the close account button on the widget.
